src/util/piped_process.cpp

```diff
@@ -88,16 +88,16 @@
 #  include <unistd.h> // library for read/write/sleep/etc. functions
 #endif
 
-#  include <cstring> // library for strerror function (on linux)
-#  include <iostream>
-#  include <vector>
+#include <util/exception_utils.h>
+#include <util/invariant.h>
+#include <util/narrow.h>
+#include <util/optional.h>
+#include <util/piped_process.h>
+#include <util/string_utils.h>
```

Comment:
Not a big deal, but we don't usually do this. Do we really need to do this here?

Reply:
No. I will revert this and replace with a clang-format commit to keep the changes in the following commit more readable.